ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,399 @@
+# Zarr Community Feedback Process - Zarr Enhancement Proposal (ZEP)
+
+--- 
+ 
+## ZEP 0 — Purpose and process
+
+Author: Sanket Verma <svsanketverma5@gmail.com>
+
+Status: Active
+
+Type: Process
+
+Created: 2022-14-03
+
+## What is ZEP?
+
+ZEP stands for Zarr Enhancement Proposal. A ZEP is a design document providing
+information to the Zarr community, describing a modification or enhancement of
+the Zarr storage specification, a new feature for its processes or environment.
```

Comment:
```suggestion
the Zarr specification, a new feature for its processes or environment.
```

Let's try to use consistent language around what we call the spec

Reply:
Agreed. I think this stems from a difference between the v2 and v3 usage. V2 uses "storage specification": https://zarr.readthedocs.io/en/stable/spec/v2.html -- I'd vote for updating to the V3 usage.

Reply:
Agreed. Having "storage" here will also be confusing when discussing stores in the spec. So "Zarr specification" seems simpler